lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -204,6 +206,9 @@ const FeatureInfoSection = React.createClass({
                                 {reactInfo.info}
                             </Otherwise>
                         </Choose>
+                        <For each="ExtraComponent" index="i" of={FeatureInfoSection.extraComponents}>
+                            <ExtraComponent key={i} viewState={this.props.viewState}/>
```

Comment:
Is just the `viewState` enough context?  It seems like most of the section's props would be useful for the extra component, too.  Especially `feature` and `catalogItem`.